packages/components/src/Popover/Popover.css

```diff
@@ -0,0 +1,83 @@
+:root {
+  --card--background-color: var(--color-white);
+  --card--base-padding: var(--space-base);
+  --popover--padding: var(--space-base);
+  --popover--width: calc(var(--base-unit) * 21.875);
+  --dismiss--button-padding: calc(var(--base-unit) / 4);
+}
+
+.header {
+  display: flex;
+  position: "relative";
+  padding: var(--popover--padding);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.dismissButton {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: var(--dismiss--button-padding);
+}
+
+.popover {
+  z-index: 1;
+  width: 100%;
+  max-width: var(--popover--width);
+  border: 1px solid var(--color-grey--light);
+  border-radius: var(--radius-base);
+  font-size: var(--base-unit);
+  line-height: normal;
+  background: var(--color-white);
+}
+
+.popover > * {
+  --public-content--padding: var(--card--base-padding);
+}
+
+.arrow,
+.arrow::before {
+  position: absolute;
+  width: var(--base-unit);
+  height: var(--base-unit);
+  background: inherit;
+}
+
+.arrow {
+  visibility: hidden;
+}
+
+.arrow::before {
+  content: "";
+  visibility: visible;
+  border: 1px solid var(--color-grey--light);
+  transform: rotate(45deg);
+  transform-origin: center;
+  clip-path: polygon(-4px -4px, 15px 0, 0 15px);
+}
+
+.popover[data-popper-placement^="top"] > .arrow {
+  bottom: -5px;
```

Comment:
Sort of pondering if we should make these sizes `calc`ed of of our sizing ones. Not sure if it would be worth it but it would be really nice to have them all tied together. 🤔

Reply:
Yea I was thinking about that too.  I don't think it's worth it right now since the arrow is so specific to popover.  Maybe if other components will use it in the future? Though I find that unlikely.